3f/pygmentize/2016-11-05.yaml

```diff
@@ -0,0 +1,9 @@
+title: Remote Code Execution
+link: https://github.com/dedalozzo/pygmentize/issues/1
+cve: ~
+branches:
+    master:
+        time: 2016-12-31 00:00:00
```

Comment:
is there a way to make users of a library aware of a serious issue even if there is not fix yet?